Gordon360/Models/ViewModels/RecIM/ParticipantViewModel.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Gordon360.Models.CCT;
+using Microsoft.Graph;
+using System;
+using System.Collections.Generic;
+
+namespace Gordon360.Models.ViewModels.RecIM
+{
+    public class ParticipantViewModel
+    {
+        public string ADUserName { get; set; }
+        public string Email { get; set; }
+        public string Role { get; set; }
+
+        public static implicit operator ParticipantViewModel(ACCOUNT a)
+        {
+            ParticipantViewModel vm = new ParticipantViewModel
+            {
+                ADUserName = a.AD_Username.Trim() ?? "",
```

Comment:
Why are you trimming here? You shouldnt have to do so on the AD_Username field

Reply:
Oh that's unfortunate. I am pretty sure neither of them need it. We don't have any AD_Usernames in the database with whitespace. @EjPlatzer ?